modules/chatops4s-examples/src/main/resources/application.conf

```diff
@@ -0,0 +1,14 @@
+slack {
+  # Get these from your Slack app configuration
+  # botToken = "xoxb-your-bot-token-here"
+  # signingSecret = "your-signing-secret-here"
+
+  # Server configuration
+  port = 3000
+  port = ${?PORT}
+}
+
+# Logging configuration
+logger {
+  level = "INFO"
```

Comment:
~~this would typically happen outside. Check Liams Pr and my comment there about Logback.~~

I see you have logback already. So what is the semantic of this value here?